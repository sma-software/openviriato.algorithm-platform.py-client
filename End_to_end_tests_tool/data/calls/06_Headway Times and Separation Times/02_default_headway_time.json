{
    "restResource": "/section-tracks/{sectionTrackId}/headway-times",
    "httpMethod": "GET",
    "urlParameters": [ "885" ],
    "queryParameters": [
        {
            "key": "fromNodeId",
            "value": "161",
            "optional": true
        },
        {
            "key": "toNodeId",
            "value": "348",
            "optional": true
        }
    ],
    "requestBody": null,
    "CSharpMethodName": "GetDefaultHeadwayTime(sectionTrackID, fromNodeID, toNodeID)",
    "CSharpMethodParameterTypes": [ "System.Int64", "System.Nullable{System.Int64}", "System.Nullable{System.Int64}" ],
    "subcategory": "Get Headway Time"
}