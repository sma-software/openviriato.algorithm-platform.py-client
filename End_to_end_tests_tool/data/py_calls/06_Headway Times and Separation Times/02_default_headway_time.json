{
  "PythonParameterMapping": {
    "section_track_id": "$.urlParameters.[0]",
    "from_node_id": "$.queryParameters.[0].value",
    "to_node_id": "$.queryParameters.[1].value"
  }
}

