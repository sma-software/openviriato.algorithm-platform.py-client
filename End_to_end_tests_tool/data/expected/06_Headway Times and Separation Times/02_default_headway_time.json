"PT2M54S"